docs/tutorials/notebooks/GWLRSinkhorn.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>if you are using the primal cost, then it's better to rename the titles to "Primal cost of LR Solution" and " ... of Entropic Solution" to avoid the ambiguity. I am also surprised that the plot of low rank is looks so much like a rank 1 or 2 matrix, can you double check?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/410/discussion/#comment-1315852322'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>